### PR TITLE
MAKE-1071, MAKE-1072: Signal processing jobs for updated and new rollups.

### PR DESCRIPTION
**Pull request by ancostas**

### CODE REVIEW

**Comment by ancostas** [comment]
I think this would handle cases of revised rollup data, as well as new data, right? Or is there another job for revised data?

**Comment by tychoish** [comment]
nope, this should catch the things.

File: units/ftdc_rollups_test.go [hunk 1]
```diff
@@ -7,6 +7,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/mongodb/amboy/queue"
+
```
[the remark this answers is not in the document]
Comment (ancostas) [reply units/ftdc_rollups_test.go]: Think I've got it now --> was auto import on the fly misbehaving.

File: units/ftdc_rollups.go [hunk 2]
```diff
@@ -149,5 +149,24 @@ func (j *ftdcRollupsJob) Run(ctx context.Context) {
...
+		Project:     result.Info.Project,
+		Variant:     result.Info.Variant,
+		Task:        result.Info.TaskName,
+		Test:        result.Info.TestName,
+		Measurement: rollup.Name,
+	}
+	processingJob := NewRecalculateChangePointsJob(id)
+	if err := j.queue.Put(ctx, processingJob); err != nil {
+		j.AddError(errors.Wrapf(err, "problem putting signal processing job %s on remote queue", j.ID()))
+		return
```
[the remark this answers is not in the document]
Comment (ancostas) [reply units/ftdc_rollups.go]: Go fmt is pushing them back apart. Or do you mean feeding the `Put` directly into `AddError`?

[the remark this answers is not in the document]
Comment (bsamek) [reply units/ftdc_rollups.go]: Yup, `j.AddError(errors.Wrap(j.queue.Put(`. If the error is nil, `errors.Wrap` returns nil, and so `j.AddError` noops.

[the remark this answers is not in the document]
Comment (tychoish) [reply units/ftdc_rollups.go]: yes, sorry, brian said what I was thinking about more clearly.

[the remark this answers is not in the document]
Comment (ancostas) [reply units/ftdc_rollups.go]: Done.

**Pull request merged**